rust/monovertex/src/metrics.rs

```diff
@@ -17,22 +17,30 @@ use tokio::task::JoinHandle;
 use tokio::time;
 use tracing::{debug, error, info};
 
+use prometheus_client::encoding::text::encode;
+use prometheus_client::encoding::EncodeLabelSet;
+use prometheus_client::metrics::counter::Counter;
+use prometheus_client::metrics::family::Family;
+use prometheus_client::metrics::gauge::Gauge;
+use prometheus_client::metrics::histogram::{exponential_buckets, Histogram};
+use prometheus_client::registry::Registry;
+
 use crate::error::Error;
 use crate::sink::SinkClient;
 use crate::source::SourceClient;
 use crate::transformer::TransformerClient;
 
 // Define the labels for the metrics
-pub const MONO_VERTEX_NAME: &str = "vertex";
+pub const MONO_VERTEX_NAME: &str = "mono_vertex_name";
```

Comment:
```suggestion
pub const MONO_VERTEX_NAME_LABEL: &str = "mono_vertex_name";
```

also should it be `mvtx`? to be consistent with what we defined in GO? `LabelMonoVertex             = "mvtx"`